src/Common/src/Interop/Windows/Crypt32/Interop.CertDuplicateCertificateContext.cs

```diff
@@ -4,13 +4,12 @@
 
 using System;
 using System.Runtime.InteropServices;
-using System.Security.Cryptography.X509Certificates;
 
 internal static partial class Interop
 {
     internal static partial class Crypt32
     {
-        [DllImport(Interop.Libraries.Crypt32, CharSet = CharSet.Unicode, SetLastError = true)]
-        internal static extern SafeCertContextHandle CertDuplicateCertificateContext(IntPtr pCertContext);
+        [DllImport(Libraries.Crypt32, CharSet = CharSet.Unicode, SetLastError = true)]
+        internal static extern IntPtr CertDuplicateCertificateContext(IntPtr pCertContext);
```

Comment:
> 17:41:07 System\Security\Cryptography\X509Certificates\X509Utils.cs(16,53): error CS0029: Cannot implicitly convert type 'System.IntPtr' to 'System.Security.Cryptography.X509Certificates.SafeCertContextHandle' [D:\j\workspace\windows-TGrou---0d2c9ac4\src\System.Windows.Extensions\src\System.Windows.Extensions.csproj]

This is a bad change.  The PKCS caller of this presumably is calling into a function that will automatically free the certificate context; which means that there are three different fixes:

1) The PKCS caller gets this into a SafeHandle, saves the handle (DangerousGetHandle()) then calls SetInvalid to prevent the SafeHandle from doing cleanup.

2) Add CertDuplicateCertificateContext_NoSafeHandle to return the IntPtr, change PKCS to call that (which then sets DllImportAttribute.EntryPoint as "CertDuplicateCertificateContext").

3) Leave this one alone, as the PKCS caller is doing "something weird" that doesn't belong in common src.

Reply:
I have problems with the second case. I'm not sure that I know how to implement the first one. So, I prepared a change for the third one